src/main/asciidoc/new-features.adoc

```diff
@@ -5,3 +5,8 @@
 === What's new in Spring Vault 1.0
 
 * Initial Vault support.
+
+[[new-features.1-1-0]]
+=== What's new in Spring Vault 1.1.0
+
+* Batch encryption & decryption support.
```

Comment:
let me know if this is proper location or if you are managing the feature list somewhere else.